pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/ElasticSearchDeleteUtil.java

```diff
@@ -54,10 +55,10 @@ public static void deleteOncHistoryDetailById(String ddpParticipantId, int oncHi
                 (List<Map<String, Object>>) ((Map<String, Object>) esDsmMap.get(ESObjectConstants.DSM)).get(
                         ESObjectConstants.ONC_HISTORY_DETAIL);
         oncHistoryDetails.removeIf(stringObjectMap -> (int) stringObjectMap.get("oncHistoryDetailId") == oncHistoryDetailId);
-        esDsmMap.put(ESObjectConstants.ONC_HISTORY_DETAIL, oncHistoryDetails);
-
+        ((HashMap<String,List<Map<String, Object>>>) esDsmMap.get(ESObjectConstants.DSM)).put(ESObjectConstants.ONC_HISTORY_DETAIL,
```

Comment:
Is there any chance that `esDsmMap.get(ESObjectConstants.DSM)` will return a null value?